	#include <sys/mman.h> only #if defined(HAVE_MMAP) (for MinGW32).

diff -ur dvipng-1.12.orig/misc.c dvipng-1.12/misc.c
--- dvipng-1.12.orig/misc.c	2009-10-01 00:39:28.000000000 +0200
+++ dvipng-1.12/misc.c	2010-02-20 18:38:56.057886522 +0100
@@ -29,7 +29,7 @@
 #define basename xbasename
 #endif
 #include <fcntl.h> /* open/close */
-#ifndef MIKTEX
+#ifdef HAVE_MMAP
 #include <sys/mman.h>
 #endif
 #include <sys/stat.h>
